### DISCUSSION TITLE
4KHDR 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#178] [模板讨论]
4KHDR 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by hecarli555** [comment]
感觉这几个站都是一伙的，4khdr也沦陷了
Failed at 7/10 request,
Fail assert: {"re": "您当前的访问请求当中含有非法字符，已经被系统拒绝", "from": "content"} from failed_asserts,
Response Error : HTTP 503: Service Unavailable,
Request URL: [https://4khdr.cn/plugin.php?id=k_misign%3Asign&operation=qiandao&formhash={{formhash|urlencode}}&format=empty&inajax=1&ajaxtarget=](https://4khdr.cn/plugin.php?id=k_misign%3Asign&operation=qiandao&formhash=%7B%7Bformhash%7Curlencode%7D%7D&format=empty&inajax=1&ajaxtarget=)

**Reply by wjf0214** [reply]
更新了，再试试。

**Comment by hecarli555** [comment]
11月28的版本更新了些什么呢？

**Reply by wjf0214** [reply]
删除了签到相关请求，因为网站去掉了签到。只保留了每日登录的。

**Comment by miaoxiaozhao** [comment]
，你已连续签到 天
签到排名：，签到等级：Lv.
积分奖励：，总天数：
金钱：4523，威望：，贡献：
用户组：高级会员，积分 4523, 距离下一级还需 477 积分

**Reply by wjf0214** [reply]
没发错的话，这不是我这最新的模板。签到早就没了。以及日志也优化了。

**Comment by hecarli555** [comment]
Failed at 2/11 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 301: Moved Permanently,
Request URL: https://4khdr.cn/member.php?mod=logging&action=login&infloat=yes&handlekey=login&inajax=1&ajaxtarget=fwin_content_login
站点貌似加了安全问题？
求大佬更新
